curio/app/src/components/App.js

```diff
@@ -1,22 +1,17 @@
 import React from 'react';
-// import { Recorder } from 'react-voice-recorder';
+// import { Recorder , myRecorder} from 'react-voice-recorder';
+import {button} from './recorder'
 // import ReactPlayer from 'react-player/youtube'
```

Comment:
I am guessing you require these commented out imports in the future? If not, then they can be removed!

Reply:
Yes, there are high chances of usage of these imports in the future. But yes, if they would be left unused they would be removed.